sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOTest.java

```diff
@@ -1678,6 +1683,23 @@ private void testWritePartition(long numTables, long numFilesPerTable, long file
     }
   }
 
+  static class IdentityDynamicTables extends DynamicDestinations<String, String> {
+
+    @Override
+    public String getDestination(ValueInSingleWindow<String> element) {
```

Comment:
Is getDestination allowed to return null? If no, better throw an UnsupportedOperationException("Not supposed to be called in this test"), or something